### PR TITLE
added bs v4. fix navigation bar since there is new implementation

### PR DESCRIPTION
**Pull request by koolfreak**

### CODE REVIEW

**Comment by koolfreak** [comment]
add bs v4 support

File: index.html [hunk 1]
```diff
@@ -47,9 +48,24 @@
...
+                <button type="button" class="btn btn-secondary">Login</button>
+                <button type="button" class="btn btn-primary">Registrieren und helfen</button>
+            </div>
+        </div>
+      </div>
+    </nav>
+    <!-- <nav class="navbar navbar-inverse navbar-fixed-top" role="navigation">
         <div class="container">
-            <!-- Brand and toggle get grouped for better mobile display -->
+            Brand and toggle get grouped for better mobile display
```
Comment (minthemiddle) [inline index.html]: Why did you comment that out? This is nothing the user needs to know...

File: index.html [hunk 2]
```diff
@@ -61,19 +77,19 @@
                     <img src="images/volunteer-planner-logo.png" alt="">
                 </a>
             </div>
-            <!-- Collect the nav links, forms, and other content for toggling -->
+            Collect the nav links, forms, and other content for toggling
```
Comment (minthemiddle) [inline index.html]: Same here. Why did you comment that out? This is nothing the user needs to know...

**Comment by minthemiddle** [comment]
Okay, that should do it.

**Pull request merged**